client/src/components/Common/selectCurrency/index.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Select } from 'antd';
+import PropTypes from 'prop-types';
+import flagOptions from './currencies';
+
+import './style.css';
+import 'antd/dist/antd.css';
+
+const { Option } = Select;
+
+const SelectCurrency = ({ className, onChange }) => (
+  <Select
+    defaultValue={
+      <div dir="ltr" className="flagSpan">
+        <span role="img" aria-label="USA">
+          🇺🇸
+        </span>
+        <div> دولار امريكي</div>
+      </div>
+    }
+    className={className}
```

Comment:
if it's just one class and you didn't want to override it, just add it above as static without using proptypes, but if you attend to fix the style or adding another style, you can add base class and then receive others as props. 
```suggestion
    className={`${className} selectCurrency`}
```